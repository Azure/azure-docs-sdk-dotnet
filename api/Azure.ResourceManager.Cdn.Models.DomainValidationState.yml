### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.Cdn.Models.DomainValidationState?alt=Azure.ResourceManager.Cdn.Models.DomainValidationState&text=DomainValidationState" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.op_Equality(Azure.ResourceManager.Cdn.Models.DomainValidationState,Azure.ResourceManager.Cdn.Models.DomainValidationState)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.op_Implicit(System.String)~Azure.ResourceManager.Cdn.Models.DomainValidationState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.op_Inequality(Azure.ResourceManager.Cdn.Models.DomainValidationState,Azure.ResourceManager.Cdn.Models.DomainValidationState)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Equals(Azure.ResourceManager.Cdn.Models.DomainValidationState)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.InternalError
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.PendingRevalidation
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.RefreshingValidationToken
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Submitting
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.TimedOut
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Unknown
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.Cdn.Models.DomainValidationState
commentId: T:Azure.ResourceManager.Cdn.Models.DomainValidationState
namespace: Azure.ResourceManager.Cdn.Models
name: DomainValidationState
fullName: Azure.ResourceManager.Cdn.Models.DomainValidationState
nameWithType: Azure.ResourceManager.Cdn.Models.DomainValidationState
assembliesWithMoniker:
- value: Azure.ResourceManager.Cdn.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Cdn v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.Cdn v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct DomainValidationState : IEquatable<Azure.ResourceManager.Cdn.Models.DomainValidationState>'
- lang: fsharp
  values:
  - value: type DomainValidationState = struct
- lang: vb
  values:
  - value: >-
      Public Structure DomainValidationState

      Implements IEquatable(Of DomainValidationState)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: Provisioning substate shows the progress of custom HTTPS enabling/disabling process step by step. DCV stands for DomainControlValidation.
metadata:
  api_name:
  - Azure.ResourceManager.Cdn.Models.DomainValidationState
  api_location:
  - Azure.ResourceManager.Cdn.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Cdn.Models.DomainValidationState
  - Azure::ResourceManager::Cdn::Models::DomainValidationState
  - DomainValidationState
  helpviewer_keywords:
  - Azure.ResourceManager.Cdn.Models.DomainValidationState struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
