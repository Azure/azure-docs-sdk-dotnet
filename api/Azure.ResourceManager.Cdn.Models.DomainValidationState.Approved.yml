### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.Cdn.Models.DomainValidationState?alt=Azure.ResourceManager.Cdn.Models.DomainValidationState&text=DomainValidationState" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
  commentId: P:Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
  name: Approved
  fullName: Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
  nameWithType: DomainValidationState.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.Cdn.Models.DomainValidationState Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.Cdn.Models.DomainValidationState'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As DomainValidationState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved*
commentId: Overload:Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
namespace: Azure.ResourceManager.Cdn.Models
name: Approved
fullName: Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
nameWithType: DomainValidationState.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.Cdn.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Cdn v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.Cdn v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
  - Azure.ResourceManager.Cdn.Models.DomainValidationState.get_Approved
  api_location:
  - Azure.ResourceManager.Cdn.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Cdn.Models.DomainValidationState.Approved
  - Azure::ResourceManager::Cdn::Models::DomainValidationState::Approved
  - Approved
  - DomainValidationState.Approved
  - DomainValidationState::Approved
  helpviewer_keywords:
  - DomainValidationState.Approved property [.NET]
  - Approved property [.NET], struct DomainValidationState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
